### ISSUE TITLE
Is it possible to use grpc options with this client library?

### ISSUE THREAD

**Issue by robinchesterman** [MichaelJCompton/Dgraph-dotnet#3]
I have a need to increase the max_recieved_message_length for GRPC. I believe options like this can be passed into the Grpc.Core Channel constructor:

```
var options = new[]
{
    new ChannelOption("grpc.max_receive_message_length", 1024 * 1024 * 1024),
};
var channel = new Channel("my/endpoint", ChannelCredentials.Insecure, options);
```

But this library abstracts that into `GRPCConnectionFactory`

Is there another way to pass options such as this?

**Comment by MichaelJCompton** [comment]
Thanks for the request.  

Not yet possible.  I'm working on a new versions.  I'll add this to the list.

**Comment by MichaelJCompton** [comment]
fixed by v0.5.2.

**Comment by MichaelJCompton** [comment]
or v0.5.3 where I publish the right build.

big changes coming for v.0.6.0, so v0.5.3 is probably the last in line before all that.
